charts/super-agent-deployment/Chart.yaml

```diff
@@ -9,7 +9,7 @@ appVersion: 0.8.0
 
 dependencies:
   - name: common-library
-    version: 1.1.1
+    version: 1.1.2
```

Comment:
[Just providing feedback on chart we own]
modifying all charts at the same time is not allowed by the CI/CD
Moreover, the version was not bumped.

---

On the other hand, this change could be quite important and I'll leave the decision to the K8s team whether we actually want to support this use case (and then "how")